binstar_client/commands/login.py

```diff
@@ -118,4 +153,6 @@ def add_parser(subparsers):
                            help="Specify your password. "
                                 "If this is not given, you will be prompted"
                            )
+    subparser.add_argument('--kerberos', action='store_true',
```

Comment:
Yeah, I was considering setting it as a configuration variable, just started with argument for ease of development.

I think the preflight request is the best option - you shouldn't be allowed to input username/password to a Kerberos-configured server.